benches/src/reuse.rs

```diff
@@ -0,0 +1,310 @@
+use std::error;
```

Comment:
I don't think it's a good idea to duplicate entire benchmark suite for this...

Reply:
Agreed. Which benches do you think we should keep?

Reply:
I think his issue is with the duplication. If we want to have the entire benchmark suite run without reusing the interpreter, we could write a macro that allows you to abstract over both. That's what I did for `smallvec`.